hq_superset/views.py

```diff
@@ -53,7 +53,8 @@ def _ucr_id_to_pks(self):
     def create_or_update(self, datasource_id):
         # Fetches data for a datasource from HQ and creates/updates a superset table
         from .oauth import get_valid_cchq_oauth_token
-        res = refresh_hq_datasource(g.hq_domain, datasource_id)
+        display_name = request.args.get("name")
```

Comment:
This sets `display_name` to the UCR's name in CommCare HQ.

(If the UCR has no name then that's a bit confusing in Superset (it will still have a UUID for a name), but it will be a bigger problem in HQ.)

Reply:
UCRs in HQ are usually named, that name gets stored in the description field of superset (since the name column has to store UCR UUID), if they don't have a name, the `display_name` will be just empty, it wouldn't run into an error.